docs_src/design/TOC.md

```diff
@@ -1,9 +1,10 @@
 # Use Cases and Design Records
 
 ## Use Case Records (UCRs)
-| Name/Link                                                                 | Short Description                                             |
-|---------------------------------------------------------------------------|---------------------------------------------------------------|
+| Name/Link                                                    | Short Description                                       |
+| ------------------------------------------------------------ | ------------------------------------------------------- |
 | [0001 System Events for Devices](./ucr/0001-System-Events-for-Devices.md) | Use Case for System Events for Device add/update/delete |
+| [0005 Record and Replay](./ucr/0005-Record-and-Replay.md)    | Use Case for Recording and Replaying event/readings     |
```

Comment:
I used 0005 for Hybrid App-Device Services: https://github.com/edgexfoundry/edgex-docs/pull/809

Reply:
Actually, I don't know why we have the numbering to begin with. Unlike ADRs, UCRs should not have a number in the file name. The UCR is meant to be identifiable by its unique name. We don't need to have two versions of the same use case, but instead would amend the original one if changes are to be made. 

ADR: https://github.com/edgexfoundry/edgex-docs/tree/main/docs_src/design/adr#naming-and-formatting
UCR: https://github.com/edgexfoundry/edgex-docs/tree/main/docs_src/design/ucr#naming-and-formatting